### DISCUSSION TITLE
Making project up to date

### DISCUSSION THREAD

**Discussion by mthnglac** [infinitered/ignite#2385] [Q&A]
I have created an application using ignite couple of months ago. Now I want my project to up to date. Is there any best practice for upgrading my packages in the project and also fetching the ignite repository last commits/features.

**Comment by frankcalise** [comment]
Hey @mthnglac thanks for your interest in Ignite. Please see the response at a similar question here: https://github.com/infinitered/ignite/discussions/2362

Open to ideas on how to make this process better if you have any!
